### PR TITLE
Password visibility switches

### PR DESCRIPTION
**Pull request by aforge**

### CODE REVIEW

File: Tinodios/Base.lproj/Main.storyboard [hunk 1]
```diff
@@ -2274,6 +2346,8 @@
         <image name="delete-message-22" width="22.5" height="22.5"/>
         <image name="edit-48" width="48" height="48"/>
         <image name="email-25" width="25" height="25"/>
+        <image name="eye.fill" catalog="system" width="64" height="38"/>
```
Comment (or-else) [inline Tinodios/Base.lproj/Main.storyboard]: warning: SF Symbol 'eye.fill' is unavailable prior to iOS 13.0. [5]

File: Tinodios/Base.lproj/Main.storyboard [hunk 2]
```diff
@@ -2274,6 +2346,8 @@
...
+        <image name="eye.slash.fill" catalog="system" width="64" height="40"/>
```
Comment (or-else) [inline Tinodios/Base.lproj/Main.storyboard]: warning: SF Symbol 'eye.slash.fill' is unavailable prior to iOS 13.0. [5]

File: Tinodios/Base.lproj/Main.storyboard (hunk 1 above)
Comment (aforge) [reply Tinodios/Base.lproj/Main.storyboard]: Thanks for pointing out. Substituted with custom icons.

**Comment by aforge** [comment]
Changes made as requested.

**Comment by aforge** [comment]
Sounds reasonable. A good yak shave :-)

**Pull request merged**